netket/experimental/dynamics/_rk_tableau.py

```diff
@@ -37,8 +37,6 @@ def _expand(x):
 
 @dataclass
 class TableauRKExplicit:
-    name: str
```

Comment:
Making me happy would be writing here a description of the Tableau with the formula we use for integration (like, a means this, b means that...